scrapers/catalog.py

```diff
@@ -118,7 +140,7 @@ def get_anchors_with_classname(element):
 
 
 def scrape_courses_from_page(courses, href):
-    '''Fills courses with course data from the href'''
+    """Fills courses with course data from the href"""
     r = requests.get(f"{BASE_URL}/{href}")
     # The "html.parser" parses pretty badly
     html = BeautifulSoup(r.content, "lxml")
```

Comment:
We'll also need to add the lxml library to requirements.txt.

```
lxml==4.9.1
```

(Otherwise we get this error.)
> bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?